elisp/core.el

```diff
@@ -217,4 +217,6 @@
                             mal-nil))
                          (t
                           mal-nil))))))
+
+    (elisp-eval . ,(mal-fn (lambda (string) (mal-string (format "%S" (eval (read (mal-value string))))))))
     ))
```

Comment:
I suggested returning a string as a starting point. Once that works, I suggest trying to convert the relevant types, as possible.

Lua suffers from a similar problem - an empty Lua table can be empty list or empty hashmap. In lua-eval I decided it's going to be an empty list.